damnit/gui/main_window.py

```diff
@@ -983,6 +992,12 @@ def run_app(context_dir, connect_to_kafka=True):
             # User said no to setting up a new database
             return 0
 
+    # configure webviewer url engine
+    global SCHEME_HANDLER, WEB_PROFILE
```

Comment:
Does `SCHEME_HANDLER` need to be global? And can't we pass `WEB_PROFILE` to `PlotlyPlot()` directly?

Reply:
right, fixed